pkg/cmd/datastore/datastore.go

```diff
@@ -469,6 +470,11 @@ func newCRDBDatastore(ctx context.Context, opts Config) (datastore.Datastore, er
 		return nil, errors.New("read replicas are not supported for the CockroachDB datastore engine")
 	}
 
+	maxRetries, err := safecast.ToUint8(opts.MaxRetries)
+	if err != nil {
+		return nil, errors.New("max-retries could not be cast to uint8")
+	}
```

Comment:
Another branch that we're unlikely to hit, but it could result in very weird behavior if we do.